integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/search/spatial/DistanceSearchProjectionIT.java

```diff
@@ -189,6 +190,25 @@ public void distanceProjection_nullUnit() {
 				.toProjection();
 	}
 
+	@Test
+	public void distanceProjection_nonProjectable() {
+		IndexSearchTarget searchTarget = indexManager.createSearchTarget().build();
+
+		SubTest.expectException( () -> {
+			searchTarget.projection().field( "nonProjectableGeoPoint", GeoPoint.class ).toProjection();
```

Comment:
As far as I can see, we already test for this in `SearchProjectionIT`. But I guess it doesn't hurt to do it here, too...